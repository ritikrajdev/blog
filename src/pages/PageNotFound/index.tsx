import React from 'react';

export default function PageNotFound() {
  return (
    <div style={{ textAlign: 'center' }}>
      <p>Uh Oh !!</p>
      <p>The thing you are looking for does not exist.</p>
    </div>
  );
}
