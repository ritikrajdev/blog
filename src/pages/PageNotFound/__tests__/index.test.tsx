import React from 'react';
import { render } from '@testing-library/react';
import PageNotFound from '..';

describe('PageNotFound', () => {
  it('should render correctly', () => {
    const screen = render(<PageNotFound />);
    expect(screen.asFragment()).toMatchSnapshot();
  });
});
