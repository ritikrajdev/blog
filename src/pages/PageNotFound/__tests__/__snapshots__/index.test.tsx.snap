// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound should render correctly 1`] = `
<DocumentFragment>
  <div
    style="text-align: center;"
  >
    <p>
      Uh Oh !!
    </p>
    <p>
      The thing you are looking for does not exist.
    </p>
  </div>
</DocumentFragment>
`;
